website/screens/components/number-input/usage/NumberInputUsagePage.tsx

```diff
@@ -0,0 +1,29 @@
+import { DxcStack, DxcText, DxcList } from "@dxc-technology/halstack-react";
+import HeadingLink from "../../../common/HeadingLink";
+import DocFooter from "../../../common/DocFooter";
+
+const NumberInputUsage = () => {
```

Comment:
Please, call it `NumberInputUsagePage` to have all the same naming.

Reply:
Done. The button component is not following this naming either. Please change it in the [PR](https://github.com/dxc-technology/halstack-style-guide/pull/729).